.teamcity/settings.kts

```diff
@@ -1,37 +1,20 @@
 import jetbrains.buildServer.configs.kotlin.v2019_2.*
 import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.script
 import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.powerShell
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.pullRequests
 import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.xmlReport
-import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.XmlReport
```

Comment:
Have you tested this importing?

Reply:
Yes, I believe that this is in play on another build configuration. @Windos suggested on another PR that these were case-sensitive, so I removed the "duplicate" entry with the incorrect casing.